workflow-controller/controller-service/src/main/java/io/hyscale/controller/util/CommandUtil.java

```diff
@@ -43,9 +45,9 @@
 	 * @param appName
 	 * @return environment name
 	 */
-	public static String getEnvName(String profile, String appName) {
+	public static String getEnvName(String profile, String appName) throws HyscaleException {
```

Comment:
Update the method along with java docs, as it no longer requires the app name, Also it doesn't need to throw hyscale exception